client/public/about.html

```diff
@@ -501,6 +501,17 @@ <h5 class="fw-bold">Camron Olsen</h5>
                 <p class="text-muted">Drexel Fall 2024</p>
               </div>
             </div>
+          </div> <div class="col-md-4">
+            <div class="card text-center mb-3 box-shadow-custom">
+              <div class="card-body p-0 pb-4">
+                <div class="mb-4">
+                  <img class="img-fluid" src="/static/img/man-user-circle-icon.svg" alt="No Profile Picture">
+                </div>
+                <h5 class="fw-bold">Lakshmipriya Rajesh Kanna</h5>
+                <div class="text-muted">MS in Software Engineering</div>
+                <p class="text-muted">Drexel University - Expected Graduation(Winter 2025)</p>
```

Comment:
Please change this to "Drexel Winter 2025" to maintain syntax of all other profiles.